pkg/apis/rke.cattle.io/v1/plan/plan.go

```diff
@@ -69,7 +73,12 @@ type File struct {
 	Path        string `json:"path,omitempty"`
 	Permissions string `json:"permissions,omitempty"`
 	Dynamic     bool   `json:"dynamic,omitempty"`
-	Minor       bool   `json:"minor,omitempty"` // minor signifies that the file can be changed on a node without having to cause a full-blown drain/cordon operation
+	// Minor signifies that the file can be changed on a node without having to cause a full-blown drain/cordon operation
+	Minor bool `json:"minor,omitempty"`
+
+	// DrainHashFunc allows a given file to insert a custom hash function as opposed to the default (hashing the content).
+	// This allows the ConfigYamlFileName to restart on "server" change, but not trigger a drain.
```

Comment:
It is a bit confusing that a file can be restarted. Could you rephrase the sentence?

Reply:
Updated.